app/controllers/vacancies_controller.rb

```diff
@@ -15,7 +15,7 @@ class VacanciesController < ApplicationController
   def index
     @filters = VacancyFilters.new(search_params)
     @sort = VacancySort.new.update(column: sort_column, order: sort_order)
-    records = Vacancy.public_search(filters: @filters, sort: @sort).page(page_number).records
+    records = Vacancy.public_search(filters: @filters, sort: @sort).page(page_number).records(includes: [:school])
```

Comment:
I failed to get bullet to fail when I reverted this change locally.

```
595 examples, 0 failures

Randomized with seed 316
```

From what you said I presume we expect this to fail here? Does it behave differently on yours?

Reply:
Good spot, it seems like only the feature specs fail, not the controller specs

Reply:
Oh wait, you ran all the specs here didn't you. That's very odd. I get:

```
Finished in 1 minute 25.32 seconds (files took 5.4 seconds to load)
595 examples, 23 failures

Failed examples:

rspec ./spec/features/job_seekers_can_search_vacancies_spec.rb:41 # Searching vacancies by keyword searchable fields #subject
rspec ./spec/features/job_seekers_can_search_vacancies_spec.rb:105 # Searching vacancies by keyword stopword search Does not include results with words removed from the index in their title
rspec ./spec/features/job_seekers_can_view_vacancies_spec.rb:22 # Viewing vacancies Only vacancies satisfying all publishing conditions are listed
rspec ./spec/features/job_seekers_can_view_vacancies_spec.rb:54 # Viewing vacancies Vacancies should paginate when over per-page limit
rspec ./spec/features/job_seekers_can_view_vacancies_spec.rb:99 # Viewing vacancies Should correctly pluralize the number of vacancies returned without a search
rspec ./spec/features/job_seekers_can_view_vacancies_spec.rb:11 # Viewing vacancies There are enough vacancies to invoke pagination
rspec ./spec/features/job_seekers_can_view_vacancies_spec.rb:43 # Viewing vacancies Vacancies should not paginate when under per-page limit
rspec ./spec/features/job_seekers_can_view_vacancies_spec.rb:81 # Viewing vacancies Should correctly pluralize the number of vacancies returned by a search
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:75 # Filtering vacancies Filterable by education phase
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:4 # Filtering vacancies Filterable by keyword
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:91 # Filtering vacancies Filterable by minimum salary
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:59 # Filtering vacancies Filterable by working pattern
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:222 # Filtering vacancies Filterable by newly qualified teacher Suitable for NQTs
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:239 # Filtering vacancies Filterable by newly qualified teacher Display all available jobs when NQT suitable is unchecked
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:108 # Filtering vacancies Filterable by maximum salary when a job's maximum salary is set
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:124 # Filtering vacancies Filterable by maximum salary when a job's maximum salary is not  set
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:282 # Filtering vacancies Searching triggers a job to write a search_event to the audit Spreadsheet correctly logs the total results when pagination is used
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:259 # Filtering vacancies Searching triggers a job to write a search_event to the audit Spreadsheet correctly logs the number of non-paginated results
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:35 # Filtering vacancies Filterable by location Search results can be filtered by the selected location and radius
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:163 # Filtering vacancies Filterable by both minimum and maximum salary when a job's salary is not within the specified salary range
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:142 # Filtering vacancies Filterable by both minimum and maximum salary when a job's salary is within the specified salary range
rspec ./spec/features/job_seekers_can_filter_vacancies_spec.rb:182 # Filtering vacancies Filterable by both minimum and maximum salary when a job's maximum salary is not set
rspec ./spec/features/job_seekers_can_sort_vacancies_spec.rb:4 # Sorting vacancies Default is to be sorted by expiry date but can be sorted by published date

Randomized with seed 4843
```

When I run all the specs. I wonder what's making it not fail for you?

Reply:
If I revert the fix I also get failures, so works as expected for me 👍 
```
595 examples, 24 failures
```